### PR TITLE
 remove closeStatement method

### PR DESCRIPTION
**Pull request by codefairy08**

### CODE REVIEW

File: sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java [hunk 1]
```diff
@@ -76,13 +72,9 @@
      * @throws SQLException failed to execute SQL, throw this exception
      */
     public static void executeUpdate(final Connection connection, final String sql, final Collection<Object> params) throws SQLException {
-        PreparedStatement preparedStatement = null;
-        try {
-            preparedStatement = connection.prepareStatement(sql);
+        try (PreparedStatement preparedStatement = connection.prepareStatement(sql);) {
```
Comment (terrymanu) [inline sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java]: Not need comma for last line of try with resource statement

File: sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java [hunk 2]
```diff
@@ -47,9 +47,7 @@
      */
     public static List<Map<String, Object>> executeQuery(final Connection connection, final String sql, final Collection<Object> params) throws SQLException {
         List<Map<String, Object>> result = new ArrayList<>();
-        PreparedStatement preparedStatement = null;
-        try {
-            preparedStatement = connection.prepareStatement(sql);
+        try (PreparedStatement preparedStatement = connection.prepareStatement(sql);) {
```
Comment (terrymanu) [inline sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java]: Not need comma for last line of try with resource statement

File: sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java [hunk 3]
```diff
@@ -95,16 +87,12 @@ public static void executeUpdate(final Connection connection, final String sql,
      * @throws SQLException failed to execute SQL, throw this exception
      */
     public static void executeBatch(final Connection connection, final String sql, final Collection<Collection<Object>> paramsCollection) throws SQLException {
-        PreparedStatement preparedStatement = null;
-        try {
-            preparedStatement = connection.prepareStatement(sql);
+        try (PreparedStatement preparedStatement = connection.prepareStatement(sql);) {
```
Comment (terrymanu) [inline sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java]: Not need comma for last line of try with resource statement

File: sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java [hunk 4]
```diff
@@ -95,16 +87,12 @@ public static void executeUpdate(final Connection connection, final String sql,
...
             for (Collection<Object> params : paramsCollection) {
```
Comment (terrymanu) [inline sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java]: variable name in for each loop should name as each

File: sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java [hunk 5]
```diff
@@ -47,9 +47,7 @@
      */
     public static List<Map<String, Object>> executeQuery(final Connection connection, final String sql, final Collection<Object> params) throws SQLException {
         List<Map<String, Object>> result = new ArrayList<>();
-        PreparedStatement preparedStatement = null;
-        try {
-            preparedStatement = connection.prepareStatement(sql);
+        try (PreparedStatement preparedStatement = connection.prepareStatement(sql);) {
             fillParamter(preparedStatement, params);
```
Comment (terrymanu) [inline sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java]: method `fillParamter` should name as `fillParamters`

File: sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java (hunk 1 above)
Comment (codefairy08) [reply sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java]: ok

File: sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java (hunk 2 above)
Comment (codefairy08) [reply sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java]: ok

File: sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java (hunk 3 above)
Comment (codefairy08) [reply sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java]: i remove it

File: sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java (hunk 4 above)
Comment (codefairy08) [reply sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java]: ok

File: sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java (hunk 5 above)
Comment (codefairy08) [reply sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java]: i remove it

**Pull request merged**